account_vat_period_end_statement/models/account.py

```diff
@@ -265,6 +265,8 @@ def _get_default_interest_percent(self):
         'Interest - Percent', default=_get_default_interest_percent)
     fiscal_page_base = fields.Integer(
         'Last printed page', required=True, default=0)
+    fiscal_year = fields.Char(
```

Comment:
Here you are adding a new functionality, I suggest you do this in a separate commit